Description: Install the .appdata.xml in /usr/share/metainfo/
Author: Mattia Rizzolo <mattia@debian.org>
Forwarded: no
Applied-Upstream: in the trunk branch
Last-Update: 2018-05-08

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -356,7 +356,7 @@
 ENDIF()
 
 #APPDATA
-SET(APPDATADIR "${CMAKE_INSTALL_DATAROOTDIR}/appdata/")
+SET(APPDATADIR "${CMAKE_INSTALL_DATAROOTDIR}/metainfo/")
 IF(NOT IS_ABSOLUTE ${APPDATADIRR})
     ADD_DEFINITIONS("-DAPPDATADIR=\\\"${CMAKE_INSTALL_PREFIX}/${APPDATADIR}\\\"")
 ELSE()
